### DISCUSSION TITLE
包引用错误但可正常运行

### DISCUSSION THREAD

**Discussion by bigthouzand** [simple-robot/Discussions#67] [Q&A]
22:36:45.874 [main] INFO  l.f.s.s.a.SimbotSpringBootBotAutoRegisterBuildConfigure - [register,153] - Registering bot with verify info [love.forte.simbot.bot.URLBotVerifyInfo@2fafa7b5]
22:36:46.937 [DefaultDispatcher-worker-3] INFO  l.f.s.m.n.578838532 - [info0,92] - ECDH key is invalid, start to fetch ecdh public key from server.
22:36:47.816 [DefaultDispatcher-worker-2] INFO  l.f.s.m.n.578838532 - [info0,92] - Successfully fetched ecdh public key from server.
22:36:48.421 [nioEventLoopGroup-5-3] ERROR l.f.s.m.n.578838532 - [error0,102] - Exception in decoding packet.
net.mamoe.mirai.internal.utils.crypto.DecryptionFailedException: null
	at net.mamoe.mirai.internal.utils.crypto.TEA.fail(TEA.kt:299)
	at net.mamoe.mirai.internal.utils.crypto.TEA.doOption$decrypt(TEA.kt:275)
	at net.mamoe.mirai.internal.utils.crypto.TEA.doOption(TEA.kt:295)
	at net.mamoe.mirai.internal.utils.crypto.TEA.decrypt(TEA.kt:319)
	at net.mamoe.mirai.internal.utils.crypto.TEA.decrypt$default(TEA.kt:318)

看了[maria库](https://github.com/mamoe/mirai/issues/2484)，说是版本问题

于是升级到v3.0.0-RC.2 + v3.0.0.0-M5
但是
 import love.forte.simboot.annotation.ContentTrim;
import love.forte.simboot.annotation.Filter;
import love.forte.simboot.annotation.Filters;
import love.forte.simboot.annotation.Listener;
import love.forte.simboot.filter.MultiFilterMatchType;
import love.forte.simbot.ID;
import love.forte.simbot.Identifies;
import love.forte.simbot.component.mirai.event.MiraiGroupMessageEvent;
import love.forte.simbot.message.*;
全标红了，但是可以运行

**Comment by ForteScarlet** [comment]
> 全标红了，但是可以运行

更新你的IDEA，或者禁用Kotlin插件
